examples/aio-sx-geo-location.yaml

```diff
@@ -0,0 +1,185 @@
+---
+apiVersion: starlingx.windriver.com/v1
+kind: DataNetwork
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: group0-data0b
+  namespace: deployment
+spec:
+  description: group0 data networks for the shared internal networks.
+  mtu: 1500
+  type: vlan
+---
+apiVersion: starlingx.windriver.com/v1
+kind: DataNetwork
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: group0-data0
+  namespace: deployment
+spec:
+  description: group0 data networks for the tenant1 networks.
+  mtu: 1500
+  type: vlan
+---
+apiVersion: starlingx.windriver.com/v1
+kind: DataNetwork
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: group0-data1
+  namespace: deployment
+spec:
+  description: group0 data networks for the tenant2 networks.
+  mtu: 1500
+  type: vlan
+---
+apiVersion: starlingx.windriver.com/v1
+kind: DataNetwork
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: group0-ext0
+  namespace: deployment
+spec:
+  description: group0 data networks for the external networks.
+  mtu: 1500
+  type: vlan
+---
+apiVersion: starlingx.windriver.com/v1
+kind: Host
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: controller-0
+  namespace: deployment
+spec:
+  match:
+    bootMAC: CONTROLLER0MAC
+  overrides:
+    location: vbox
+  profile: controller-profile
+---
+apiVersion: starlingx.windriver.com/v1
+kind: HostProfile
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: controller-profile
+  namespace: deployment
+spec:
+  administrativeState: unlocked
+  bootDevice: /dev/disk/by-path/pci-0000:00:0d.0-ata-1.0
+  console: tty0
+  installOutput: text
+  interfaces:
+    ethernet:
+    - class: platform
+      name: lo
+      platformNetworks:
+      - mgmt
+      - cluster-host
+      port:
+        name: lo
+    - class: platform
+      name: oam0
+      platformNetworks:
+      - oam
+      port:
+        name: enp0s3
+    - class: data
+      dataNetworks:
+      - group0-data0
+      - group0-data0b
+      - group0-ext0
+      name: data0
+      port:
+        name: eth1000
+    - class: data
+      dataNetworks:
+      - group0-data1
+      name: data1
+      port:
+        name: eth1001
+  labels:
+    openstack-compute-node: enabled
+    openstack-control-plane: enabled
+    openvswitch: enabled
+    sriov: enabled
+  personality: controller
+  processors:
+  - functions:
+    - count: 0
+      function: vswitch
+    node: 0
+  provisioningMode: dynamic
+  rootDevice: /dev/disk/by-path/pci-0000:00:0d.0-ata-1.0
+  storage:
+    osds:
+    - function: osd
+      path: /dev/disk/by-path/pci-0000:00:0d.0-ata-2.0
+    volumeGroups:
+    - name: nova-local
+      physicalVolumes:
+      - path: /dev/disk/by-path/pci-0000:00:0d.0-ata-1.0
+        size: 4
+        type: partition
+  subfunctions:
+  - controller
+  - worker
+---
+apiVersion: starlingx.windriver.com/v1
+kind: System
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: geo-location
```

Comment:
The file you're generating is putting your latitude/longitude tags in a separate section from the other System because your "name" doesn't match that of the default Section, similar to how you previously had a unique namespace. Please update the name to align with the default Section "vbox", regenerate the file, and ensure your latitude and longitude settings are contained in the same System section as other settings.